demos/blast/blast_in_example.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+#include <seqan/blast.h>
+
+using namespace seqan;
+
+int main()
+{
+    typedef Align<String<AminoAcid>, ArrayGaps> TAlign;
+    typedef BlastMatch<CharString, CharString, uint32_t, TAlign> TBlastMatch;
```

Comment:
Please change all types from (u)intXX_t to __(u)intXX - see e.g. https://github.com/seqan/seqan/blob/master/include/seqan/platform/platform_gcc.h